### DISCUSSION TITLE
Disable hit sounds when using cinema mod

### DISCUSSION THREAD

**Discussion by gi-b716** [ppy/osu#28491] [Ideas] [answered]
Disable hit sounds when using cinema mod, as it looks weird

https://github.com/ppy/osu/assets/78394473/d6464fa5-6bd8-40eb-9322-a923e7841d2a

**Comment by gi-b716** [comment]
Especially when you are listening to the music/storyboard of the map, the percussion sound effects are a bit redundant

**Comment by bdach** [comment, marked as answer]
stable doesn't do this so I don't think we are going to change this. A workaround would be to change effect volume to 0 I guess.

**Comment by gi-b716** [comment]
NP
---- Replied Message ----
| From | Bartłomiej ***@***.***> |
| Date | 6/16/2024 01:56 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [ppy/osu] Disable hit sounds when using cinema mod (Discussion #28491) |

stable doesn't do this so I don't think we are going to change this. A workaround would be to change effect volume to 0 I guess.

**Comment by shiumano** [comment]
Hmmm...would it be too much to add "disable hit sounds" to the mod settings?

**Reply by bdach** [reply]
There's no need to add a separate setting for it. Again: change effect volume to 0.
